plugin/src/main/kotlin/de/triplet/gradle/play/internal/PlayPublishPackageBase.kt

```diff
@@ -0,0 +1,52 @@
+package de.triplet.gradle.play.internal
+
+import com.google.api.services.androidpublisher.AndroidPublisher
+import com.google.api.services.androidpublisher.model.LocalizedText
+import com.google.api.services.androidpublisher.model.Track
+import com.google.api.services.androidpublisher.model.TrackRelease
+import java.io.File
+
+abstract class PlayPublishPackageBase : PlayPublishTaskBase() {
+    protected fun AndroidPublisher.Edits.updateTracks(
+            editId: String,
+            inputFolder: File,
+            releaseStatus: String,
+            versions: List<Long>,
+            trackType: String,
+            userPercent: Double
+    ) {
+        val track = tracks()
+                .list(variant.applicationId, editId)
+                .execute().tracks?.firstOrNull { it.track == trackType } ?: Track()
+
+        val releaseText = if (inputFolder.exists()) {
```

Comment:
Let's use plural here because this holds a list of LocalizedText objects, right?